content/intro-to-storybook/svelte/es/creating-addons.md

```diff
@@ -0,0 +1,478 @@
+
+### Crear el componente de contenido
+
+Hemos completado nuestro primer objetivo. Es hora de comenzar a trabajar en el segundo.
+
+Para completarlo, necesitamos hacer algunos cambios en nuestras importaciones e introducir un nuevo componente que muestre la información del recurso.
+
+Realice los siguientes cambios en el archivo de complemento:
+
+```javascript
+//.storybook/design-addon/register.js
+import React, { Fragment } from 'react';
+/* same as before */
+import { useParameter } from '@storybook/api';
+
+const Content = () => {
+  const results = useParameter('assets', []); // story's parameter being retrieved here
+  return (
+    <Fragment>
+      {results.length ? (
+        <ol>
+          {results.map(i => (
+            <li>{i}</li>
+          ))}
+        </ol>
+      ) : null}
+    </Fragment>
+  );
+};
+```
+
+Creamos el componente, modificamos las importaciones, todo lo que falta es conectar el componente a nuestro panel y tendremos un complemento funcional capaz de mostrar información relativa a nuestras historias.
+
+Su código debería tener el siguiente aspecto:
+
+```javascript
+//.storybook/design-addon/register.js
+import React, { Fragment } from 'react';
+import { AddonPanel } from '@storybook/components';
+import { useParameter } from '@storybook/api';
+import { addons, types } from '@storybook/addons';
+
+const Content = () => {
+  const results = useParameter('assets', []); // story's parameter being retrieved here
+  return (
+    <Fragment>
+      {results.length ? (
+        <ol>
+          {results.map(i => (
+            <li>{i}</li>
+          ))}
+        </ol>
+      ) : null}
+    </Fragment>
+  );
+};
+
+addons.register('my/design-addon', () => {
+  addons.add('design-addon/panel', {
+    title: 'assets',
+    type: types.PANEL,
+    render: ({ active, key }) => (
+      <AddonPanel active={active} key={key}>
+        <Content />
+      </AddonPanel>
+    ),
+  });
+});
+```
+
+Tenga en cuenta que estamos usando el [useParameter](https://storybook.js.org/docs/addons/api/#useparameter), este práctico gancho nos permitirá leer la información proporcionada por la opción `parameters` para cada historia, que en nuestro caso será una ruta única a un recuros o una lista de rutas. Lo verás en efecto en breve.
+
+### Usando nuestro complemento con una historia
+
+Hemos conectado todas las piezas necesarias. Pero, ¿cómo podemos ver si realmente funciona y muestra algo?
+
+Para hacerlo, haremos un pequeño cambio en el archivo `task.stories.ts` y agregaremos la opción[parameters](<https://storybook.js.org/docs/configurations/options-parameter/#per-story-options>).
```

Comment:
@IvanSotelo sorry for the "fumble fingering" (pardon the bad pun) but the filename should be "task.stories.js" instead of "task.stories.ts" and so should be the comment in the code snippet. Once this goes live i'll make a follow up pr to fix this.